react/EXPERIMENTAL_Table.ts

```diff
@@ -1,7 +1,5 @@
 import Table from './components/EXPERIMENTAL_Table/index'
-import useTableState from './components/EXPERIMENTAL_Table/hooks/useTableState'
 import { TABLE_DENSITIES } from './components/EXPERIMENTAL_Table/constants'
 
 export default Table
-export { useTableState }
 export { TABLE_DENSITIES }
```

Comment:
The `TABLE_DENSITIES` should be exported in the same way that the `useTableState` is.

Reply:
Solved on another PR